prime-router/build.gradle.kts

```diff
@@ -573,7 +573,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion")
     implementation("org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.0")
```

Comment:
This PR is all configuration changes _except_ this one line. 
 
[Release notes](https://github.com/Kotlin/kotlinx.coroutines/blob/master/CHANGES.md) says there are some possible break changes. Not sure if they impact anything but maybe save this one line change for a different PR?

Reply:
@TomNUSDS I looked at those and I think we will be fine. It passed testing locally so I'm not super worried about it.